src/core/services/layout/layout.js

```diff
@@ -136,63 +163,84 @@
      * @param {boolean=} addDirectiveAsClass
      */
     function attributeWithObserve(className, addDirectiveAsClass) {
-      return function() {
+
+      return ['$$mdLayout', '$document', '$parse', '$interpolate', function(_$$mdLayout_, $document, _$parse_, _$interpolate_) {
```

Comment:
You should be able to use `@ngInject` here instead of manually writing out the injection array.